### ISSUE TITLE
Cannot locate elements

### ISSUE THREAD

**Issue by sherif191** [microsoft/WinAppDriver#1525]
I have issue that driver cannot locate elements after logging to system successfully (only can locate element by accessibility Id , but  almost all element didn't have accessibility Id)

Note : I tried to locate element by name,xpath and localized control type (tag name) ,but it didn't work with me

**Comment by anunay1** [comment]
Is inspect.exe showing the elements.
